packages/blockchain/src/consensus/casper.ts

```diff
@@ -20,11 +18,6 @@ export class CasperConsensus implements Consensus {
 
   public async validateConsensus(): Promise<void> {}
 
-  public async validateDifficulty(header: BlockHeader): Promise<void> {
-    if (header.difficulty !== BIGINT_0) {
-      const msg = 'invalid difficulty.  PoS blocks must have difficulty 0'
-      throw new Error(`${msg} ${header.errorStr()}`)
-    }
-  }
+  public async validateDifficulty(): Promise<void> {}
```

Comment:
Where is the `difficulty` must be 0 check now?

Reply:
Hey there,
thanks a lot for the review, all good points I guess, will address one by one:

So this one originated from a former more radical version, where I fully deleted the `CasperConsensus`, and as an in-between step deleted this difficulty check. This is not really a consensus check but just a simple format check and is in fact also done in the BlockHeader: https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/block/src/header.ts#L502

I think it is beneficial to remove it here since it otherwise "tricks" us into thinking we *would* do some kind of consensus validation here for PoS (must be, there is some code! 😋 ), while in reality this is not the case (PoS consensus validation is done on the CL side).

No super strong opinion, but I would tend to keep this removed.

Reply:
I should have checked upon reviewing because I suspected this check was in our Block Header as well (this would actually be the more logical place!). I agree it should not be in blockchain. This is actually a great change/removal, let's keep it :smile: :+1: